### PR TITLE
Update Firefox versions for GlobalEventHandlers API

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR updates and corrects the real values for Firefox and Firefox Android for the `GlobalEventHandlers` API, based upon results from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v1.1.7).  Results are manually confirmed for accuracy.

Tests Used: https://mdn-bcd-collector.appspot.com/tests/api/GlobalEventHandlers

### CODE REVIEW

File: api/GlobalEventHandlers.json [hunk 1]
```diff
@@ -947,10 +947,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "3.5"
+              "version_added": "9"
```
Comment (foolip) [inline api/GlobalEventHandlers.json:950]: There's a lot of changes from 3.5. Can you check where that data came from? Maybe it was supported on one of the other interfaces?

Comment (vinyldarkscratch) [reply api/GlobalEventHandlers.json:950]: This data came from the initial wiki migration.  Doing some additional testing around, I couldn't find any interface that supported these event handlers, so it's probably that the event is supported in Firefox 3.5, but not the event handlers.

Comment (foolip) [reply api/GlobalEventHandlers.json:950]: Yes, that's a common situation.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
And that was the last PR for the year, local time :)

**Comment by vinyldarkscratch** [comment]
Happy New Years, @foolip!  I've still got 9 hours to go myself.  😛

**Comment by foolip** [comment]
A belated happy new year to you too, @vinyldarkscratch!

**Pull request merged**